lively.freezer/src/bundler.js

```diff
@@ -753,17 +764,40 @@ export default class LivelyRollup {
       // can stub it with our FreezerRuntime
       runtimeCode += SYSTEMJS_STUB;
     }
+    if (this.isResurrectionBuild) {
+      runtimeCode += 'lively.isResurrectionBuild = true;';
+    }
     runtimeCode += regeneratorSource;
     runtimeCode += polyfills;
     return es5Transpilation(runtimeCode);
   }
 
+  generateAssetPreloads () {
+    return arr.compact(arr.uniq(this.projectAssets.map(asset => {
+      // also preload all of the used fonts in the bundle
+      if (asset.newName?.endsWith('.mp4')) {
+        return `<link rel="preload" href="${joinPath('./assets', `${asset.newName}`)}" as="video">`;
+      }
+      if (asset.newName?.endsWith('.mp3')) {
```

Comment:
Absolutely. Since you work on the asset browser right now anyways, we can proceed whatever feels best to you.
I am open to extending this branch, or moving the above stuff over to yours, since its not really essential anyways to the fast load.

Reply:
Looking at this again, I propose leaving this for now. We have an export of supported formats for images, as that is what the asset browser currently supports. Once we know how first class support for audio and videos should look we can proceed and touch this as well. I have a mental note on it.